### DISCUSSION TITLE
Migration from System.Drawing to Microsoft.Maui.Graphics

### DISCUSSION THREAD

**Discussion by Lukasl32** [Shane32/QRCoder#373] [Q&A]
As Microsoft mentioned in this post ([System.Drawing just for Windows](https://docs.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only)), System.Drawing will no longer be developed other than on Windows (I think they were more likely to maintain but no longer develop). This would severely limit the some key functionality of this library to Windows only.
Since I am considering using this sophisticated library in my projects, I would like to know if development will continue and if it is planned to move to a multiplatform library such as Microsoft.Maui.Graphics or some other alternative?

**Comment by julealgon** [comment]
Related:
- https://github.com/codebude/QRCoder/issues/315
